in_toto/user_settings.py

```diff
@@ -0,0 +1,132 @@
+"""
+<Program Name>
+  user_settings.py
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Oct 25, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Provides methods to read environment variables and rcfiles and to write
+  them to the in_toto.settings module (override them).
+
+  Notes:
+  - Variable values are converted to lists if they contain colons
+  - Variable names are converted to upper case
```

Comment:
To be precise, do you mean that in-toto settings are always upper case?

Can you be a bit more precise here, perhaps providing an example?  Are `in-toto.settings.SOME_SETTING` settings always upper case regardless of the way that environments variables are written (i.e., upper case, lower case, or mixed case) ?